demo/witness/README.md

```diff
@@ -6,22 +6,11 @@ A Cloudflare Worker to act as a witness for Rebase claims.
 
 Requires [Wrangler](https://developers.cloudflare.com/workers/cli-wrangler/install-update#additional-installation-instructions).
 
-You need a Cloudflare account, and the Account ID needs to go in
-the `wrangler.toml`.
+You need a Cloudflare account, and the Account ID needs to go in the `wrangler.toml`.
```

Comment:
should probably provide a wrangler.toml example that people should then cp and fill in the necessary fields

Reply:
There is one! `wrangler.toml.example`